code/_onclick/drag_drop.dm

```diff
@@ -47,4 +47,65 @@ actually if you do something in that proc like changing user location or whateve
 to inform the game this action was expected and its fine
 */
 /atom/proc/MouseDrop_T(atom/dropping, mob/user, params) // return TRUE if you want to prevent us click the object after it
-	return
+	return FALSE
+
+/client/MouseDown(datum/object, location, control, params)
+	if(QDELETED(object)) // Yep, you can click on qdeleted things before they have time to nullspace. Fun.
+		return
```

Comment:
This might be worth a TM for possible perf concerns

Reply:
I've gone and profiled this, it's acceptable. Won't even hit a second of time in total during a round